test/users.js

```diff
@@ -0,0 +1,159 @@
+const supertest = require("supertest");
+const assert = require("assert");
+const {models} = require("../sequelize");
+const app = require("../express/routes");
+
+//routes tests
+describe("GET /", () => {
+    it("should have status code 200", (done) => {
+        supertest(app)
+            .get("/")
+            .expect(200)
+            .end((err, res) => {
+                if (err) done(err);
+                done();
+            });
+    });
+});
+
+describe("GET /users", () => {
+    it("should return all users and have status code 200", (done) => {
+      supertest(app)
+            .get("/users")
+            .expect(200)
+            .end((err, res) => {
+                if (err) done(err);
+                done();
+            });
+    });
+});
+
+describe("GET /users/:id", () => {
+    it("should return user and have status code 200", async () => {
+        const testUser = await models.user.findOne();
+
+        const res = supertest(app)
+            .get(`/users/${testUser.id}`)
+            .expect(200);
+    });
+});
+
+describe("POST /users", () => {
+    it("should create user and have status code 201", async () => {
+        const newTestUser = {user: {
+            userName: "sunbeam",
+            email: "v.lykov@gmail.com",
+            telephone: "+375 44 757-53-96",
+            password: "44444",
+            firstName: "Slava",
+            lastName: "Lykov"
+        }};
+
+        const numberOfUsersBefore = await models.user.count();
+
+        const res = await supertest(app)
+            .post("/users")
+            .send(newTestUser)
+            .expect(201);
+
+        const numberOfUsersAfter = await models.user.count();
+        const userById = await models.user.findByPk(res.body.user.id);
+
+        assert.equal(numberOfUsersBefore + 1, numberOfUsersAfter, "Should create user");
+        assert.equal(newTestUser.user.firstName, userById.firstName, "Should create correct user");
+    });
+});
+
+describe("PUT /users/:id", () => {
+    it("should update user and have status code 200", async () => {
+        const newUserName = "newUserName";
+        const newEmail = "newEmail@gmail.com";
+
+        const testUserBefore = await models.user.findOne();
+        testUserBefore.userName = newUserName;
+        testUserBefore.email = newEmail;
+
+        const editTestUser = {user: testUserBefore.toJSON()};
+
+        const res = await supertest(app)
+            .put(`/users/${testUserBefore.id}`)            
+            .send(editTestUser)
+            .expect(200);
+
+        const testUserAfter = await models.user.findByPk(testUserBefore.id);
+        
+        assert.equal(testUserAfter.userName, newUserName, "Should update userName");
+        assert.equal(testUserAfter.email, newEmail, "Should update email");
+    });
+});
+
+describe("DELETE /users/", () => {
+    it("should create user, delete him and have status code 200", async () => {
```

Comment:
you are not testing user creation here. Only user deletion. Please correct the description